README.md

```diff
@@ -7,6 +7,8 @@
   - [Build the CLI from Sources](#build-the-cli-from-sources)
   - [Generating Build-Info](#generating-build-info)
     - [Go](#go)
+    - [Maven](#mvn)
+    - [Gradle](#gradle)
```

Comment:
I think that abive 3 items - 
```
    - [Go](#go)
    - [Maven](#mvn)
    - [Gradle](#gradle)
```
are a bit of an "overkill" for 3 liner sections. Let's remove them.